### ISSUE TITLE
Grammar mistake

### ISSUE THREAD

**Issue by LeeColi16** [lainsce/quilter#99]
There is an issue with the latest version of Quilter for elementary OS. On the Markdown Cheatsheet, under Links & Special, there is a grammar mistake. "This creates an horizontal rule" should be "This creates a horizontal rule.

Please note that I am using British English, though I do not know if this will affect the mistake. If it is not present in American English, it will be in British English. Please see the attached screenshot for details.
![screenshot from 2017-11-07 21 55 21](https://user-images.githubusercontent.com/33467902/32520000-74930990-c406-11e7-9ced-cd05e664690f.png)

**Comment by lainsce** [comment]
You could make a British English translation for Quilter, and do the necessary fixes, it's correct in American English.

**Comment by LeeColi16** [comment]
OK. I am not entirely sure how I would go about doing that as the only programming language I know is a bit of Commodore BASIC. Would it be relatively straightforward?

**Comment by lainsce** [comment]
It's very simple :+1: : 
- Click the fork button here on the Quilter main page, the one with the tuning fork
- Open a Terminal, and clone the forked repository on your PC with `git clone https://github.com/<your_github_name_here>/quilter`
- On the po folder, open the *.pot file with a program called POEdit (you can install it by writing: `sudo apt install poedit` on the Terminal).
- Once it's open, the program will guide you into making a new translation.
- Then, your en_GB.po file is done, put it into the po folder
- Then, open a terminal inside the folder quilter that you've cloned earlier.
- Write:
  - `git checkout -b "translation-en-gb"` (This makes a branch for your translation)
  - `git add *` (This adds all the files you've made into the branch)
  - `git push` (This pushes the changes to your fork)

Now, the final stretch :running_man: :
- In my Quilter repo, click on the Pull Request tab
- Push the green New Pull Request button
- Mark your branch going to master
- Click on the Create Pull Request button

And voilá, like magic, I'll be seeing your translation and I'll merge the files from it to my repo! :smiley:

**Comment by LeeColi16** [comment]
OK. I'll do that now. :-)

**Comment by LeeColi16** [comment]
`git push origin translation-en-gb` didn't work properly; see the error below:
```remote: Permission to lainsce/quilter.git denied to LeeColi16.
fatal: unable to access 'https://github.com/lainsce/quilter/': The requested URL returned error: 403

**Comment by lainsce** [comment]
> Just wondering if the .mo file that POEdit created is required, or if it can be deleted.

It can be deleted

> git push origin translation-en-gb didn't work properly;

You need to push your changes to your fork, did you clone from your fork or from my repo?
You need to first clone the quilter repo by pushing the button here on the repo with "<tuning_fork_icon> Fork", and then type `git clone https://github.com/LeeColi16/quilter` before starting things.

**Comment by lainsce** [comment]
I know git is hard, but once you do it the first time, the next times will be a joy

**Comment by lainsce** [comment]
By the way, the tuning fork icon Fork button is this one:
![Imgur](https://i.imgur.com/MYfKWum.png)

**Comment by Melyanna** [comment]
Question - would @LeeColi16 also need to add the languag code to the LINGUAS file?
(As I would like to add an Italian translation, if that is OK, I would have to follow the same process)

**Comment by lainsce** [comment]
Yes. :+1: 
Also there's a README file inside the po folder that explains the process further.

**Comment by Melyanna** [comment]
Thanks! I just added Italian.
I think your instructions above might be missing one step, after "git add *" shouldn't the user also commit changes?

**Comment by lainsce** [comment]
Ah yeah, I forgot! :stuck_out_tongue:

**Comment by lainsce** [comment]
@LeeColi16 Do you have any stuff to add to this bug, such as questions? If not, I'll close it since you already got the hang of translations.
